projects/storefrontapp-e2e-cypress/cypress/helpers/site-context-selector.ts

```diff
@@ -38,6 +38,7 @@ export const TITLE_REQUEST = `${Cypress.env(
   'OCC_PREFIX'
 )}/${CONTENT_CATALOG}/titles?lang=${LANGUAGE_EN}&curr=${CURRENCY_USD}`;
 
+export const CONTEXT_URL_EN_USD = `/${CONTENT_CATALOG}/${LANGUAGE_EN}/${CURRENCY_USD}`;
```

Comment:
Every other const here seems to start with `${BASE_URL}` or `..OCC_PREFIX..`. Maybe we should stick to this "convention"?